### PR TITLE
add honcho start and modified the config.py to match wishlistdb

### LINKED ISSUE #26: Add honcho functionality
**User Story:**
As a developer, I need honcho start, so that we can start the service fast

**Assumptions:**
* Vagrantfile is set up
* #42 is completed

**Acceptance Criteria:**
AC1) Service is started when type ```honcho start```
AC2) Make sure command ```honcho``` works the ```Procfile``` added in #42 

**Testing Plan:**
Service is started when type ```honcho start```

### PR DESCRIPTION
**Pull request by ZiyiHuang96**
testing process:
`vagrant destroy`
`vagrant up`
`cd /vagrant/`
`honcho start`

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -116,7 +116,7 @@ Vagrant.configure(2) do |config|
   ######################################################################
   config.vm.provision "shell", inline: <<-SHELL
     # uncomment this line if you want to run flask app after vagrant booted
-    sudo -H -u vagrant sh -c '. ~/venv/bin/activate && cd /vagrant && FLASK_APP=service flask run --host=0.0.0.0'
+    # sudo -H -u vagrant sh -c '. ~/venv/bin/activate && cd /vagrant && FLASK_APP=service flask run --host=0.0.0.0'
```
Comment (biegel) [inline Vagrantfile:119]: how does the app start through honcho?  don't we need a command in the `Vagrantfile` to start the app after dependencies are downloaded

Comment (ZiyiHuang96) [reply Vagrantfile:119]: By using honcho start command @biegel

Comment (ZiyiHuang96) [reply Vagrantfile:119]: @biegel I fixed the merge conflict, and when you do a vagrant destroy and up, you can use `honcho start` command to start the service under the `/vagrant/` folder fast, and `ctrl-c` to stop

Comment (ZiyiHuang96) [reply Vagrantfile:119]: @biegel @billykern @VGmt Can someone test this PR? I think you need to `vagrant destory` first and `vagrant up` to make it work. Then `vagrant ssh`, `cd /vagrant/` and `honcho start`.

Comment (billykern) [reply Vagrantfile:119]: hmm is there possibly some issue with the DB setup? i just tried those instructions...

```
➜  wishlists git:(honcho-start) vagrant ssh
Welcome to Ubuntu 20.04.2 LTS (GNU/Linux 5.10.25-linuxkit x86_64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
(venv) vagrant@ubuntu:~$ cd vagrant
-bash: cd: vagrant: No such file or directory
(venv) vagrant@ubuntu:~$ cd /vagrant/
(venv) vagrant@ubuntu:/vagrant$ honcho start
/usr/lib/python3.8/subprocess.py:848: RuntimeWarning: line buffering (buffering=1) isn't supported in binary mode, the default buffer size will be used
  self.stdout = io.open(c2pread, 'rb', bufsize)
03:52:27 system | web.1 started (pid=17391)
03:52:27 web.1  | [2021-07-27 03:52:27 +0000] [17393] [INFO] Starting gunicorn 20.0.4
03:52:27 web.1  | [2021-07-27 03:52:27 +0000] [17393] [INFO] Listening at: http://0.0.0.0:5000 (17393)
03:52:27 web.1  | [2021-07-27 03:52:27 +0000] [17393] [INFO] Using worker: sync
03:52:27 web.1  | [2021-07-27 03:52:27 +0000] [17395] [INFO] Booting worker with pid: 17395
03:52:28 web.1  | [2021-07-27 03:52:28 +0000] [INFO] [__init__] Logging handler established
03:52:28 web.1  | [2021-07-27 03:52:28 +0000] [INFO] [__init__] **********************************************************************
03:52:28 web.1  | [2021-07-27 03:52:28 +0000] [INFO] [__init__] ********  W I S H L I S T S   S E R V I C E   R U N N I N G  *********
03:52:28 web.1  | [2021-07-27 03:52:28 +0000] [INFO] [__init__] **********************************************************************
03:52:28 web.1  | [2021-07-27 03:52:28 +0000] [CRITICAL] [__init__] (psycopg2.OperationalError) FATAL:  database "wishlistdb" does not exist
03:52:28 web.1  |
03:52:28 web.1  | (Background on this error at: http://sqlalche.me/e/13/e3q8): Cannot continue
03:52:28 web.1  | [2021-07-27 03:52:28 +0000] [INFO] [glogging] Worker exiting (pid: 17395)
03:52:28 web.1  | Setting up logging for service...
03:52:28 web.1  | [2021-07-27 03:52:28 +0000] [17393] [INFO] Shutting down: Master
03:52:28 web.1  | [2021-07-27 03:52:28 +0000] [17393] [INFO] Reason: App failed to load.
03:52:28 system | web.1 stopped (rc=4)
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+COPY config.py .
+COPY service ./service
+
+# Expose any ports the app is expecting in the environment
+ENV PORT 5000
+EXPOSE $PORT
+
+ENV GUNICORN_BIND 0.0.0.0:$PORT
+ENTRYPOINT ["gunicorn"]
+CMD ["--log-level=info", "service:app"]
```
Comment (billykern) [inline Dockerfile:30]: do we still need this Dockerfile too?
https://github.com/NYU-DevOps-Squad-Wishlists/wishlists/blob/main/.devcontainer/Dockerfile

**Pull request closed without merging**